charts/ai-service/templates/deployment.yaml

```diff
@@ -61,6 +61,21 @@ spec:
           {{- end }}
       {{- end }}
       {{- end }}
+      {{- if and .Values.artifactsCache.enabled .Values.artifactsCache.artifacts }}
+        - name: {{ .Values.artifactsCache.downloader.name }}
+          image: {{ .Values.artifactsCache.downloader.image }}
+          command:
+            - /files/download-artifacts.sh
```

Comment:
i am not sure, it cooooooooould be that this file is not executable, we would have to check or test